db/migrate/20220321091220_add_personal_owner_to_teams.rb

```diff
@@ -0,0 +1,25 @@
+class AddPersonalOwnerToTeams < ActiveRecord::Migration[6.1]
+  def up
+    add_column :teams, :type, :string, null: false
+
+    Team.all.find_each do |team|
```

Comment:
do this with update_all or even SQL ... it's way more performant that way.
read doc if you do not understand the difference. 
there's a encryptable migration where we did the same